src/convert_image/x86.rs

```diff
@@ -16,6 +16,38 @@
 
 use crate::convert_image::common::*;
 
+#[cfg(target_arch = "x86")]
+use core::arch::x86::_bswap;
```

Comment:
Is there a particular reason to import from core crate?
Is it possible to import from:
`std::arch`

like for the other conversion functions?

Reply:
By looking at rust docs it seems that _bswap functions are available only in core::arch, there is a bswap function in std::intrinsics but is experimental